app/build.gradle

```diff
@@ -63,4 +65,5 @@ dependencies {
         implementation "org.jetbrains.anko:anko-$name:$ankoVersion"
     }
     implementation project(':onlineapi')
+    implementation files('build/libs/YouTubeAndroidPlayerApi.jar')
```

Comment:
You sure this is the correct way to include this lib?
Probably move the jar to libs folder and import it from there.

Reply:
@the-dagger bhaiya it is already in the libs folder there is some issue with the jetifier